packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -16,35 +16,52 @@ export enum TabsVariant {
 }
 
 export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivElement>, 'onSelect'> {
-  /** content rendered inside the Tabs Component. */
+  /** Content rendered inside the tabs component. */
   children: React.ReactNode;
-  /** additional classes added to the Tabs */
+  /** Additional classes added to the tabs */
   className?: string;
-  /** the index of the active tab */
+  /** The index of the active tab */
   activeKey?: number | string;
-  /** handle tab selection */
+  /** Callback to handle tab selection */
   onSelect?: (event: React.MouseEvent<HTMLElement, MouseEvent>, eventKey: number | string) => void;
-  /** uniquely identifies the Tabs */
+  /** Uniquely identifies the tabs */
   id?: string;
-  /** enables the filled tab list layout */
+  /** Enables the filled tab list layout */
   isFilled?: boolean;
-  /** enables Secondary Tab styling */
+  /** Enables secondary tab styling */
   isSecondary?: boolean;
-  /** Aria-label for the left Scroll Button */
+  /** Enables box styling to the tab component */
+  isBox?: boolean;
+  /** Enables vertical tab styling */
+  isVertical?: boolean;
+  /** Aria-label for the left scroll button */
   leftScrollAriaLabel?: string;
-  /** Aria-label for the right Scroll Button */
+  /** Aria-label for the right scroll button */
   rightScrollAriaLabel?: string;
-  /** determines what tag is used around the Tabs. Use "nav" to define the Tabs inside a navigation region */
+  /** Determines what tag is used around the tabs. Use "nav" to define the tabs inside a navigation region */
   variant?: 'div' | 'nav';
```

Comment:
do we normally use `component` where you specify the type of element to be used?